SnackOverflow/src/main/java/ro/utcn/sd/vasi/SnackOverflow/data_assembler/AnswerAssembler.java

```diff
@@ -0,0 +1,28 @@
+package ro.utcn.sd.vasi.SnackOverflow.data_assembler;
+
+import ro.utcn.sd.vasi.SnackOverflow.model.Answer;
+import ro.utcn.sd.vasi.SnackOverflow.repository.api.AnswerRepository;
+import ro.utcn.sd.vasi.SnackOverflow.repository.api.RepositoryFactory;
+
+import java.util.List;
+
+public class AnswerAssembler extends GeneralAssembler<Answer, AnswerRepository> implements AnswerRepository {
```

Comment:
As we discussed: a little over-engineered with the assemblers, but it is not that bad after all.